crashlogger.js

```diff
@@ -23,9 +23,9 @@ let hadException = false;
  * Logs when a crash happens to console, then e-mails those who are configured
  * to receive them.
  *
- * @param {Error} err
+ * @param {NodeJS.ErrnoException} err
```

Comment:
`NodeJS.ErrnoException` extends `Error` to include `errno`, `code`, `path`, and `syscall` properties, since `Error` only includes `stack`. It's a `@types/node` specific thing though. It's there since `repl.js` was passing the same type of error to the crashlogger. I can change it to `(Error | NodeJS.ErrnoException)` or back to `Error` and make `tsc` ignore the lines that complain about having the wrong type of error